aspnetcore/test/integration-tests.md

```diff
@@ -78,7 +78,7 @@ Infrastructure components, such as the test web host and in-memory test server (
 
 The `Microsoft.AspNetCore.Mvc.Testing` package handles the following tasks:
 
-* Copies the dependencies file (*\*.deps*) from the SUT into the test project's *bin* folder.
+* Copies the dependencies file (*\*.deps*) from the SUT into the test project's *bin* directory.
```

Comment:
Was there a 🧀 move on me? "Folder" was welcome from the start in docs. Was it determined that "directory" is more xplat friendly? If so, that makes sense. I'll start going that way.

Reply:
The official guidance is:

"Use directory only in content for developers when you need to refer to the structure of the file system or to match the API."

Since this is developer content, and we're talking about file system structure, I used *directory*.

Reply:
They may have changed it ... my copy is old. My old version has another line there before it says that. It says this leading into that .....

> Most of the time, use folder instead. 

There are *many* examples in the official guidance of them using "folder."

e.g ....

> 5.5.22	child folder
> Don't use. Use subfolder, subdirectory, folder, or the name of the folder.

> 5.6.21	destination
> Use a more precise term, such as website or folder, to describe an end point, such as the location reached when a customer clicks a hyperlink, the folder to which a file is copied or moved, or the document in which a linked or embedded object is stored. Use destination as a modifier only if necessary for clarity.

The reason that I said it might be better for xplat is that I recalled this remark ...

> 5.8.20	folder, folder icon
> In general, try not to talk about UI. Instead, talk about what the customer should do. 
In content about Windows, use folder to refer to a container for files and other folders. Use directory only in content for developers when you need to refer to the structure of the file system or to match the API. For MS-DOS, use directory. 

... it's the little bit there about "In content about Windows."

This is a break from what we've been doing. Let me know if it's a new repo rule ... "folder" is all over the place to the tune of **_>700 instances of the word_** across the docs.